src/modules/launcher/PowerLauncher/Helper/NativeMethods.cs

```diff
@@ -36,6 +36,9 @@ internal static class NativeMethods
         [DllImport("user32.dll")]
         internal static extern int SetWindowLong(IntPtr hWnd, int nIndex, int dwNewLong);
 
+        [DllImport("user32.dll")]
+        internal static extern int SetWindowLong(IntPtr hWnd, int nIndex, long dwNewLong);
```

Comment:
I know what the motivation for the inclusion of this import is, but I think we shouldn't risk doing it. On native Windows, `LONG` is 32 bits, and in C# it's 64.
Also, see my comment in WindowsInteropHelper.cs.